Flick/Cells/ListTableViewCell.swift

```diff
@@ -113,6 +113,7 @@ class ListTableViewCell: UITableViewCell {
             make.top.equalTo(titleLabel.snp.bottom).offset(padding)
             make.leading.trailing.equalToSuperview()
             make.bottom.equalToSuperview().inset(padding)
+            make.height.equalTo(120)
```

Comment:
Does this cause like constraint errors? since we also went top and bottom constraint. we usually pick one or the other right

Reply:
it doesn't cause constraint error from what I could see. you can see my reply in the other comment :>